### PR TITLE
Add -h command line argument

### PR DESCRIPTION
**Pull request by maboiteaspam**
It displays help from the Usage section of
the README file.
If stdout is detected as piped, it will display
raw MD content.
Updated Usage section of README file.

### CODE REVIEW

File: changelog-maker.js [hunk 1]
```diff
@@ -51,6 +58,16 @@ function replace (s, m) {
   return s
 }
 
+function showUsage () {
+  var usage = fs.readFileSync('README.md', 'utf8')
+    .replace(/[\s\S]+(## Usage\n[\s\S]*)\n## [\s\S]+/m, '$1')
+  if (process.stdout.isTTY)
```
Comment (rvagg) [inline changelog-maker.js]: I usually make a habit of putting braces around an `if` if the lines within it are >1, even if it's a single statement, no big deal here but just FYI

**Comment by rvagg** [comment]
lgtm

**Comment by LestaD** [comment]
:+1:

**Comment by TooTallNate** [comment]
:+1:

**Comment by rvagg** [comment]
hm, thanks @TooTallNate, I obviously dropped the ball on this, merged (heavily modified sorry, it's quite out of sync) and published

**Pull request closed without merging**